client/state/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import thunkMiddleware from 'redux-thunk';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import sharing from './sharing/reducer';
+import sites from './sites/reducer';
+import ui from './ui/reducer';
+
+/**
+ * Module variables
+ */
+const reducer = combineReducers( {
+	sharing,
+	sites,
+	ui
+} );
+
+export function createReduxStore() {
```

Comment:
That is correct, but we have:
```
194 matches across 84 files
```
Not every single one of this 84 files is a component, but I'm guessing that would mean connecting most of the components in the whole of calypso.
I can do that, but do we want to reduxify it so much?